### PR TITLE
User Story 60501: cdn-with-custom-origin

### PR DESCRIPTION
**Pull request by TomArcherMsft**
Part of POC to test generating sample code and articles using OpenAI.

### CODE REVIEW

**Comment by TomArcherMsft** [comment]
@grayzu @lonegunmanb This sample is ready for review.

**Review by lonegunmanb — APPROVED** [review]
Thanks @TomArcherMsft for opening this pr! LGTM, but I'd like to ask @grayzu for a further review.

File: quickstart/101-cdn-with-custom-origin/readme.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+- [azurerm_cdn_profile](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/cdn_profile)
+- [azurerm_cdn_endpoint](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/cdn_endpoint)
+
+## Variables
+
+| Name | Description | Default |
+|-|-|-|
+| `resource_group_name_prefix` | Prefix of the resource group name that's combined with a random ID so name is unique in your Azure subscription. | rg |
+| `resource_group_location` | Location of the resource group. | eastus |
+| `origin_url` | Url of the origin. | null |
```
Comment (grayzu) [inline quickstart/101-cdn-with-custom-origin/readme.md]: Looks like you do have a default for this.

File: quickstart/101-cdn-with-custom-origin/outputs.tf
Comment (grayzu) [inline quickstart/101-cdn-with-custom-origin/outputs.tf:1]: Should probably output the CDN endpoint FQDN as well.

**Review by grayzu — CHANGES_REQUESTED** [review]
Looking Fantastic!

Couple of minor comments.

**Comment by TomArcherMsft** [comment]
@grayzu @lonegunmanb I've made the fixes.

**Comment by grayzu** [comment]
Kicking off tests. If they pass and @lonegunmanb does not have any issues, this should be good to merge.

**Pull request merged**